server/src/helpers/transactionHelpers/transactionHelpers.ts

```diff
@@ -63,3 +70,48 @@ export const createOrdinaryEvent = (
 
   return transaction;
 };
+
+export const deleteTransaction = (
+  transaction: TransactionInterface,
```

Comment:
Better convention for interfaces would be ITransaction instead of TransactionInterface ( this is for all interfaces e.g. ICat, IPerson, IUser)